### ISSUE TITLE
Cross compilation detected. Use PKG_CONFIG_ALLOW_CROSS=1

### ISSUE THREAD

**Issue by tritone11** [cross-rs/cross#250]
Hello, I am trying to compile to aarch64-unknown-linux-gnu, and i get this when it's compiling the hidapi crate
```
$ PKG_CONFIG_ALLOW_CROSS=1 OPENSSL_STATIC=1 cross build --target aarch64-unknown-linux-gnu
   Compiling hidapi v0.4.1
error: failed to run custom build command for `hidapi v0.4.1`
process didn't exit successfully: `/target/debug/build/hidapi-76bd7f33aee12772/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to find libusb-1.0: "Cross compilation detected. Use PKG_CONFIG_ALLOW_CROSS=1 to override"', src/libcore/result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by Dylan-DPC** [comment]
Thanks.

**Comment by Alexhuszagh** [comment]
Closed in #386.
